Non-profits.md

```diff
@@ -14,6 +14,12 @@ There are also some non-profits that share some or all of their code.
 * https://github.com/geneontology/noctua
 * https://github.com/geneontology/helpdesk
 
+## [MRC Centre for Genomics and Global Health (CGGH)](https://www.sanger.ac.uk/collaboration/mrc-centre-genomics-and-global-health-cggh/)
+* https://github.com/cggh/scikit-allel
```

Comment:
this is good, but I'd prefer we link to the organization instead of a specific repo: so https://github.com/cggh would be the links